framework/include/material_image_helpers.hpp

```diff
@@ -375,6 +400,27 @@ namespace gvk
 		return result;
 	}
 
+	/**	This is a convenience method that allows to compile a selection of shared owning models and mesh indices.
+	 *	Valid usage means passing a model as parameter, and following it up with one or multiple mesh index parameters.
+	 *
+	 *	Model parameters must have shared ownership enabled.
```

Comment:
Is that true? Because if so, we wouldn't require the `avk::shared(aModel)` ccall in line #364 above, I think. 
Isn't it rather the case, that passed models are put into "shared" state when they are used with this convenience function?

Reply:
Ah yes, wrong assumption on my end. I thought shared() would throw an error when it is not enabled.